CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Renamed the project name from `github.com/coreos/mantle` to `github.com/flatcar-linux/mantle` ([#241](https://github.com/flatcar-linux/mantle/pull/241))
 - Default server on Equinix Metal ([#256](https://github.com/flatcar-linux/mantle/pull/256), [#257](https://github.com/flatcar-linux/mantle/pull/257))
 - Azure: reworked resource cleanup to rely on automatic NIC/IP/OS disk removal ([#271](https://github.com/flatcar-linux/mantle/pull/271))
+- Remove `--repo-branch` option from cork ([](https://github.com/flatcar-linux/mantle/pull/))
```

Comment:
```suggestion
- Remove `--repo-branch` option from cork ([#283](https://github.com/flatcar-linux/mantle/pull/283))
```

Reply:
Done.
And I have just realized, I added the changelog line to a wrong place. Fixed that as well.

Reply:
In the CHANGELOG format, we have the `REMOVED` section, I think this is the right place for this PR. :)

Reply:
For example: https://github.com/flatcar-linux/mantle/blob/flatcar-master/CHANGELOG.md#removed

Reply:
Done moving to the `Removed` section, thanks.